clar2wasm/tests/wasm-generation/regression.rs

```diff
@@ -80,3 +80,20 @@ fn default_to_2() {
         })))
     );
 }
+
+#[test]
+fn foo() {
```

Comment:
- `foo` is not the best name here.
- this test should be moved next to the other tests for the filter function.
- this test should use `crosscheck` and fail when we have an unexpected result.
- the example function could be simplified.

Reply:
renamed